projects/addon-doc/README.md

```diff
@@ -101,12 +101,12 @@ npm i @taiga-ui/addon-doc
     ```typescript
     import {NgModule} from '@angular/core';
     import {RouterModule} from '@angular/router';
-    import {generateRoutes, TUI_DOC_PAGE_MODULES} from '@taiga-ui/addon-doc';
```

Comment:
let's also deprecate TUI_DOC_PAGE_MODULES constant and replace it in all samples in demo

Reply:
I have to change absolutely in all places (there it is in very many components, etc.)?